python/ray/serve/schema.py

```diff
@@ -487,6 +517,118 @@ def get_empty_schema_dict() -> Dict:
         return {"applications": []}
 
 
+@PublicAPI(stability="alpha")
+class DeploymentDetails(BaseModel, extra=Extra.forbid):
+    name: str = Field(description="Deployment name.")
+    deployment_status: DeploymentStatus = Field(
+        description="The current status of the deployment."
+    )
+    message: str = Field(
+        description=(
+            "If the deployment status is UNHEALTHY, this will describe the issue in "
```

Comment:
Could you update this docstring? The `message` is sometimes included even when the deployment is not `UNHEALTHY` ([example](https://github.com/ray-project/ray/blob/3f1a880f7a4b248366ca00ea31483341d2eee1cf/python/ray/serve/_private/deployment_state.py#L1620-L1624)).

Reply:
Thanks! Changed to `If there are issues with the deployment, this will describe the issue in more detail.`